book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] I'll suggest you to implement the the method to add rentals as below so that the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. take a look at the code below:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```